internal/gengapic/gengapic.go

```diff
@@ -70,6 +70,19 @@ type options struct {
 	transports        []Transport
 }
 
+// Utility function for stringifying the Transport enum
+func (t Transport) String() string {
+	switch t {
+	case grpc:
+		return "grpc"
+	case rest:
+		return "rest"
+	default:
```

Comment:
If we currently don't have other transport types and don't plan to support arbitrary ones, can we return one of the defined values by default?

Reply:
This is a string representation of the current value, so unless we change the value to be valid (which we should do elsewhere, when we initially read it from the flag), I would keep the current code as correct.